hard-protocol-home.js

```diff
@@ -144,6 +144,53 @@ const commonDenomMapper = (denom) => {
   return formattedDenom;
 }
 
+const cache = new Map(); 
+const ibcDenomMapper = async (denom) => {
+  if (!denom.includes("ibc")) return;
+  if (cache.has(denom)) {
+    return cache.get(denom);  
+  };
+  const request = await fetch(BASE_URL + '/ibc/apps/transfer/v1/denom_traces/' + denom.replace("ibc/",""));
+  const ibcDenom = await request.json();
+  cache.set(denom, ibcDenom.denom_trace.base_denom);
+  return ibcDenom.denom_trace.base_denom; 
+}; 
+
+const normalizeDenoms = async (denomsList) => {
+  const readable = [];
```

Comment:
```suggestion
  const readableDenoms = [];
```
and when used elsewhere similar to your naming convention for `readableParams`